.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+on: [pull_request]
+jobs:
+  static_analysis:
+    name: Static analysis with phpstan
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: |
+          echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
```

Comment:
caching could be simplified with https://github.com/ramsey/composer-install

Reply:
Good call, thanks!